tests/platform_tests/link_flap/test_cont_link_flap.py

```diff
@@ -0,0 +1,133 @@
+            candidates = build_test_candidates(duthost, fanouthosts, 'unknown')
+
+            if not candidates:
+                pytest.skip("Didn't find any port that is admin up and present in the connection graph")
+
+            for dut_port, fanout, fanout_port in candidates:
+                toggle_one_link(duthost, dut_port, fanout, fanout_port, watch=True)
+
+        # Wait 60 secs so that BGP routes are relearned
+        time.sleep(60)
+
+        # Record memory status at end
+        memory_output = duthost.shell("show system-memory")["stdout"]
+        logging.info("Memory Status at end: %s", memory_output)
+
+        # Record orchagent CPU utilization at end
+        orch_cpu = duthost.shell("show processes cpu | grep orchagent | awk '{print $9}'")["stdout"]
+        logging.info("Orchagent CPU Util at end: %s", orch_cpu)
+
+        # Record Redis Memory at end
+        end_time_redis_memory = duthost.shell("redis-cli info memory | grep used_memory_human | sed -e 's/.*:\(.*\)M/\\1/'")["stdout"]
+        logging.info("Redis Memory at start: %s M", start_time_redis_memory)
+        logging.info("Redis Memory at end: %s M", end_time_redis_memory)
+
+        # Calculate diff in Redis memory
+        incr_redis_memory = float(end_time_redis_memory) - float(start_time_redis_memory)
+        logging.info("Redis absolute  difference: %d", incr_redis_memory)
+
+        # Check redis memory only if it is increased else default to pass
+        if incr_redis_memory > 0.0:
+            percent_incr_redis_memory = (incr_redis_memory / float(start_time_redis_memory)) * 100
+            logging.info("Redis Memory percentage Increase: %d", percent_incr_redis_memory)
+            pytest_assert(percent_incr_redis_memory < 5, "Redis Memory Increase more than expected: {}".format(percent_incr_redis_memory))
+
+        # Record ipv4 route counts at end
+        end_time_ipv4_route_counts = duthost.shell("show ip route summary | grep Total | awk '{print $2}'")["stdout"]
+        logging.info("IPv4 routes at start: %s", start_time_ipv4_route_counts)
+        logging.info("IPv4 routes at end: %s", end_time_ipv4_route_counts)
+
+        # Make Sure all ipv4 routes are relearned with jitter of ~5
+        incr_ipv4_route_counts = abs(int(float(start_time_ipv4_route_counts)) - int(float(end_time_ipv4_route_counts)))
+        pytest_assert(incr_ipv4_route_counts < 5, "Ipv4 routes are not equal after link flap")
+
+        end_time_ipv6_route_counts = duthost.shell("show ipv6 route summary | grep Total | awk '{print $2}'")["stdout"]
+        logging.info("IPv6 routes at start: %s", start_time_ipv6_route_counts)
+        logging.info("IPv6 routes at end: %s", end_time_ipv6_route_counts)
+
+        incr_ipv6_route_counts = abs(int(float(start_time_ipv6_route_counts)) - int(float(end_time_ipv6_route_counts)))
+        pytest_assert(incr_ipv6_route_counts < 5, "Ipv6 routes are not equal after link flap")
+
+        # Wait for 15 more secs.
+        time.sleep(15)
```

Comment:
Can you add this 15 seconds to  the next wait_until timeout?

Reply:
Done. Thanks